.github/workflows/daily-e2e-tests-ibmcloud.yaml

```diff
@@ -4,15 +4,13 @@
 # Build and push container images for each cloud provider.
 ---
 name: daily e2e tests for ibmcloud
+
 on:
   schedule:
     # Runs "at 05:00(UTC time) every day" (see https://crontab.guru)
     # will base on default branch `main`
     - cron: '0 5 * * *'
-  push:
```

Comment:
I think we still need the push target? @liudalibj?

Reply:
@jtumber-ibm thanks JT's mention.

The push target is for develop/debug purpose, as the pipeline already there and the workflow works as our expected (although the test result is not stable :( ), we can remove the push target now.

Reply:
You can always use the new `workflow_dispatch` trigger to run a workflow (on your fork, or on the main repo) if you need to debug a workflow, see https://github.com/confidential-containers/cloud-api-adaptor/pull/1171#discussion_r1260640254